### ISSUE TITLE
Error when saving an edit to a redirect

### ISSUE THREAD

**Issue by henrypenny** [Dolphiq/craft3-plugin-redirect#110]
When editing an existing redirect I get an SQL error.

SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'uid' cannot be null
The SQL being executed was: UPDATE `elements` SET `dateUpdated`='2020-04-07 02:53:12', `uid`=NULL WHERE `id`=17387

It looks like the uid field is being updated with NULL.

I'm using: "version": "1.0.23"

**Comment by jonlongnecker** [comment]
Same issue here. I can't save an existing redirect without an Internal Server Error and the same error in the logs about `uid cannot be null.`

**Comment by jonlongnecker** [comment]
Checking back in here - I really need to update an existing redirect and can't. Is this add-on still being maintained?

**Comment by hilbertknl** [comment]
Same problem, hoping for a fix to come out soon!

**Comment by hilbertknl** [comment]
I created a pull request for this issue #113

**Comment by jonlongnecker** [comment]
Yeah it's been two months with no reply so assuming this has been abandoned. I don't think it's even in the Craft Plugin store anymore. Unfortunately I just ended up buying Retour to get things working again.
